src/main/java/org/folio/des/service/impl/JobServiceImpl.java

```diff
@@ -187,6 +187,18 @@ public void deleteOldJobs() {
     deleteJobs(jobsToDelete);
   }
 
+  @Transactional
+  @Override
+  public void resendExportedFile(UUID jobId) {
+    org.folio.des.domain.dto.Job job = get(jobId);
+    if (job.getFiles().isEmpty()) {
+    throw new NotFoundException(String.format("The exported file is missing for jobId: %s", job.getId()));
```

Comment:
the same issue with formatting, need to add 2 spaces before throw statement

Reply:
fixed